docs/api-reference/apps/v1beta2/definitions.html

```diff
@@ -7260,7 +7253,7 @@ <h3 id="_any">any</h3>
 </div>
 <div id="footer">
 <div id="footer-text">
-Last updated 2017-08-08 01:49:22 UTC
+
```

Comment:
@janetkuo Seems weird here that this line is removed after running auto-generated.